### DISCUSSION TITLE
How can I prevent VirtualBox app from starting after reboot?

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [i3/i3#5063] [Q&A] [answered]
First, thanks for this actually awesome wm!

So now, whenever I reboot, i3 works *awesome*, just one little thing that *for some reason* VirtualBox app also starts. I have no idea why and I don't have it in start up, proof:

![image](https://user-images.githubusercontent.com/31178401/181595233-a95cd48a-d05a-46d9-a6c0-6b945069c005.png)

What am I missing here?

**Comment by Airblader** [comment, marked as answer]
What about your i3 config? Maybe it's launched in there. It goes without saying that i3 doesn't randomly launch apps, so I'm not sure what makes you think i3 is involved here.

**Reply by WhiteBlackGoose** [reply]
> It goes without saying that i3 doesn't randomly launch apps.

Absolutely, haha. But I found nothing about it in the config. I used the automatically generated one, and I myself made just *one* change to it in the end, unrelated to virtual box.
[Here](https://gist.github.com/WhiteBlackGoose/5696872895d543e6eadbf27f8cec583d) is the gist (from `~/.config/i3/config`)

**Reply by Airblader** [reply]
How do you launch i3? Maybe VirtualBox has some option for automatically launching on boot?

**Reply by WhiteBlackGoose** [reply]
I added `i3` to Session and Startup -> Autostart, following some random tutorial. 
Forgot to mention, I'm on xubuntu 22.04.

I didn't find anything inside virtual box, and the reason why I'm asking on this repo is because it started happening only after I installed i3 (not blaming it ofc, but it somehow affected it, right? can't say)

**Reply by WhiteBlackGoose** [reply]
Actually it may be because of sessions. The tutorial I followed recommended saving session after disabling `xdesktop` and `xfwm4`, can it be because of that?

P. S. yeah, the saved session has virtual box in it. Let me try to do something with it

**Reply by WhiteBlackGoose** [reply]
Okay, I'm an idiot. Fix: reboot, do the shenanigans with sessions without starting any app, save it. Otherwise some running programs get into it. Thanks, haha
